### PR TITLE
update Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb

### PR DESCRIPTION
**Pull request by wangyems**
**Description**: Describe your changes.
1 convert tf model using tf2onnx as we deprecated the keras2onnx 
2 remove all OMP threads related as ORT adopts eigen thread pool as default
3 using the most updated packages

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/python/tools/transformers/notebooks/Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb [hunk 1]
```diff
@@ -316,75 +317,85 @@
...
     "optimized_model.use_dynamic_axes()\n",
     "optimized_model.save_model_to_file(optimized_model_path)"
-   ]
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "failed in shape inference <class 'AttributeError'>\n"
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/notebooks/Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb:383]: Failed here. May need take a look.

**Comment by tianleiwu** [comment]
Instead of "bert-base-cased", I suggest we use fine tuned model "deepset/roberta-base-squad2"  or "deepset/bert-base-cased-squad2" or "bert-large-uncased-whole-word-masking-finetuned-squad" to run this notebook.

File: onnxruntime/python/tools/transformers/notebooks/Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb (hunk 1 above)
Comment (wangyems) [reply onnxruntime/python/tools/transformers/notebooks/Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb:383]: "GlobalAveragepool" and "Reciprocal" is currently not supported in shape inference tool

**Comment by wangyems** [comment]
> Instead of "bert-base-cased", I suggest we use fine tuned model "deepset/roberta-base-squad2" or "deepset/bert-base-cased-squad2" or "bert-large-uncased-whole-word-masking-finetuned-squad" to run this notebook.

I prefer to hold off as the result between tf and onnx have big difference, Need to let tf2onnx team take a look

File: onnxruntime/python/tools/transformers/notebooks/Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb (hunk 1 above)
Comment (wangyems) [reply onnxruntime/python/tools/transformers/notebooks/Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb:383]: it seems there are more issues besides this two unsupported op.

**Comment by wangyems** [comment]
> Instead of "bert-base-cased", I suggest we use fine tuned model "deepset/roberta-base-squad2" or "deepset/bert-base-cased-squad2" or "bert-large-uncased-whole-word-masking-finetuned-squad" to run this notebook.

done

File: onnxruntime/python/tools/transformers/notebooks/Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb [hunk 2]
```diff
@@ -0,0 +1,729 @@
...
+    "del session"
+   ],
+   "outputs": [
+    {
+     "output_type": "error",
+     "ename": "KeyboardInterrupt",
+     "evalue": "",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/notebooks/Tensorflow_Tf2onnx_Bert-Squad_OnnxRuntime_CPU.ipynb:425]: This cell output is not normal. Maybe re-run the notebook could fix it.

**Review by tianleiwu — APPROVED** [review]

**Pull request merged**